src/compiler/transformers/declarations.ts

```diff
@@ -1140,6 +1140,9 @@ namespace ts {
                         const varDecl = factory.createVariableDeclaration(newId, /*exclamationToken*/ undefined, resolver.createTypeOfExpression(input.expression, input, declarationEmitNodeBuilderFlags, symbolTracker), /*initializer*/ undefined);
                         errorFallbackNode = undefined;
                         const statement = factory.createVariableStatement(needsDeclare ? [factory.createModifier(SyntaxKind.DeclareKeyword)] : [], factory.createVariableDeclarationList([varDecl], NodeFlags.Const));
+
+                        preserveJsDoc(statement, input);
+                        removeAllComments(input);

```

Comment:
this is probably a question for @rbuckton , but why do you need to `removeAllComments(input)`? The other uses of preserveJSDoc don't do that.